nautobot/dcim/templates/dcim/device.html

```diff
@@ -6,295 +6,304 @@
-                                            {% if object.primary_ip4 %}
-                                                <a href="{% url 'ipam:ipaddress' pk=object.primary_ip4.pk %}">{{ object.primary_ip4.address.ip }}</a>
-                                                {% if object.primary_ip4.nat_inside %}
-                                                    <span>(NAT for {{ object.primary_ip4.nat_inside.address.ip }})</span>
-                                                {% elif object.primary_ip4.nat_outside %}
-                                                    <span>(NAT: {{ object.primary_ip4.nat_outside.address.ip }})</span>
-                                                {% endif %}
-                                            {% else %}
-                                                <span class="text-muted">&mdash;</span>
-                                            {% endif %}
-                                        </td>
-                                    </tr>
-                                    <tr>
-                                        <td>Primary IPv6</td>
-                                        <td>
-                                            {% if object.primary_ip6 %}
-                                                <a href="{% url 'ipam:ipaddress' pk=object.primary_ip6.pk %}">{{ object.primary_ip6.address.ip }}</a>
-                                                {% if object.primary_ip6.nat_inside %}
-                                                    <span>(NAT for {{ object.primary_ip6.nat_inside.address.ip }})</span>
-                                                {% elif object.primary_ip6.nat_outside %}
-                                                    <span>(NAT: {{ object.primary_ip6.nat_outside.address.ip }})</span>
-                                                {% endif %}
-                                            {% else %}
-                                                <span class="text-muted">&mdash;</span>
-                                            {% endif %}
-                                        </td>
-                                    </tr>
-                                    {% if object.cluster %}
-                                        <tr>
-                                            <td>Cluster</td>
-                                            <td>
-                                                {% if object.cluster.group %}
-                                                    <a href="{{ object.cluster.group.get_absolute_url }}">{{ object.cluster.group }}</a> /
-                                                {% endif %}
-                                                <a href="{{ object.cluster.get_absolute_url }}">{{ object.cluster }}</a>
-                                            </td>
-                                        </tr>
-                                    {% endif %}
-                                </table>
+<div class="row">
```

Comment:
Can you please revert the indentation changes you've made to this file and keep the diffs to just what has functionally changed instead?

Reply:
Sure thing. VS Code auto format made the updates.